ob-pattern-1/templates/obam/obam-deployment.yaml

```diff
@@ -0,0 +1,93 @@
+# Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: wso2-obam-deployment
+  namespace: wso2
+spec:
+  replicas: 1
+  minReadySeconds: 30
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+    type: RollingUpdate
+  selector:
+    matchLabels:
+      deployment: wso2-obam
+  template:
+    metadata:
+      labels:
+        deployment: wso2-obam
+    spec:
+      containers:
+      - name: wso2-obam
+        image: docker.wso2.com/wso2-obam:1.4.0
```

Comment:
The tags like imageName should be taken from the Values.yaml. 
e.g : refer https://github.com/wso2/kubernetes-is/blob/master/advanced/is-pattern-1/templates/is/identity-server-deployment.yaml

Reply:
We have not started parameterizing the variables yet. First we decided to create static files, commit them and then move on to parameterize the values. That why this has not been taken from the values.yaml

Reply:
Fixed with https://github.com/wso2/kubernetes-open-banking/pull/13/commits/7b8a303ec096ec71e64859f2f84e46d23732a9e3